content/introductions/reaping-rewards/index.tsx

```diff
@@ -6,14 +6,15 @@ export default function ReapingRewards({ lang }) {
 
   return (
     <Introduction lang={lang}>
-      <Text className="pt-2">{t('reaping_rewards.paragraph_one')}</Text>
-      <Text className="pt-2">{t('reaping_rewards.paragraph_two')}</Text>
+      <Text className="text-lg md:text-xl">{t('reaping_rewards.paragraph_one')}</Text>
+      <Text className="text-lg md:text-xl mt-4">{t('reaping_rewards.paragraph_two')}</Text>
+      <Text className="text-lg md:text-xl mt-4">{t('reaping_rewards.paragraph_three')}</Text>
     </Introduction>
   )
 }
 
 export const metadata = {
   title: 'reaping_rewards.title',
   subtitle: `reaping_rewards.subtitle`,
```

Comment:
With these values now being not necessary to render the template they should no longer be needed in the metadata. I'm fairly sure we could always add these values through each individual index for future content

Reply:
Fixed, thanks.